src/sfizz/FilterPool.cpp

```diff
@@ -36,6 +36,17 @@ void sfz::FilterHolder::setup(const Region& region, unsigned filterId, int noteN
         fast_real_distribution<float> dist { 0.0f, description->random };
         baseCutoff *= centsFactor(dist(Random::randomGenerator));
     }
+
+    auto cutoffMod = description->cutoffMod;
+    for (const auto& mod : description->cutoffCC) {
+        const auto& curve = resources.getCurves().getCurve(mod.data.curve);
+        const float value = resources.getMidiState().getCCValue(mod.cc);
+        cutoffMod += curve.evalNormalized(value) * mod.data.modifier;
+    }
+
+    baseCutoff *= centsFactor(cutoffMod);
+    baseCutoff = Default::filterCutoff.bounds.clamp(baseCutoff);
```

Comment:
I think we can clamp once in the end, would you agree?

Reply:
If you're proposing we consoldate the lines into one, that could work. The main reason I wrote it like that was to make it more legible.